README.md

```diff
@@ -20,238 +20,13 @@ The core responsibilities of Hagall are:
-* `503 Service Unavailable` means that HDS reaches a reverse proxy (such as nginx), but the backend server (Hagall) is unavailable. Check the reverse proxy logs to verify that HDS actually reaches the reverse proxy you think it reaches. And make sure Hagall is running and that the reverse proxy points to Hagall using the address it listens to (port 4000 by default). You can also try to restart your reverse proxy.
-* `400 Bad Request` could mean that the format of your public endpoint is invalid. It needs to be prefixed with `https://` and not `http://`. Follow the same format as in the [configuration](#Configuration) section. Note that if you use the Docker Compose setup, `VIRTUAL_HOST` and `LETSENCRYPT_HOST` should be without the `https://` prefix and `HAGALL_PUBLIC_ENDPOINT` should have the `https://` prefix.
-
-#### Admin port
-
-Other than the ordinary port used for WebSocket traffic, Hagall is also listening on a separate port for administrative purposes.
-This port is not exposed externally by default and should not be, so you would have to connect to it from the same machine as you're running Hagall on or from your internal network only.
-
-```
-    --admin-addr                string               Admin listening address.
-                                                     Env:     HAGALL_ADMIN_ADDR
-                                                     Default: ":18190"
-```
-
-For example, if Hagall is running on your local machine and you visit http://localhost:18190/debug/pprof/ you can get profiling data.
-If you set up Prometheus, you can scrape metrics from the /metrics endpoint.
-
-##### Endpoints
-
-* `/metrics` - Prometheus-formatted metrics
-* `/health` - Health check endpoint, returns 200 OK if service is running
-* `/debug/pprof/` - Index page of Go's [pprof](https://pkg.go.dev/net/http/pprof) package
-  * There are many sub-endpoints useful for profiling. Please refer to the index page
-
-#### Metrics
-
-Other than the standard Golang metrics coming from [prometheus/client_golang](https://github.com/prometheus/client_golang) with `go_*`, `process_*` prefixes etc., these additional metrics are available:
-
-* `ws_*` - WebSocket related metrics
-  * A very useful metric to look at is `ws_connected_clients`, which is a gauge that represents the number of connected WebSocket clients. Note that the smoke tests are also WebSocket clients, so it will flip between 0 and 1 during these tests.
+## Documentations
+
+- [Video Tutorial](doc/videotutorial.md)
+- [Server Operator's Manual](doc/operatormanual.md)
+- [Minimum Requirements](doc/minimumrequirements.md)
+- [Deployment](doc/deployment.md)
+- [Testing](doc/testing.md)
+- [Troubleshooting](doc/troubleshooting.md)
+- [Admin Endpoints](doc/adminendpoints.md)
+- [Hagall Entity Component System](doc/entitycomponentsystem.md)
```

Comment:
I think `docs` is a more common folder name, perhaps use that? Also I think we can add hyphens to the file names, they aren't .go files :D
For example `operator-manual.md` and `minimum-requirements.md`.

Reply:
renamed to `docs` & added dashes.